dicom_utils/dicom.py

```diff
@@ -49,6 +49,10 @@
 }
 
 
+# Used to indicate preprocessed DICOMs
+ALGORITHM_PRESENTATION_TYPE: Final = "FOR MEDCOG ALGORITHM"
```

Comment:
The length needs to be <= 16
```suggestion
ALGORITHM_PRESENTATION_TYPE: Final = "FOR ALGORITHM"
```